rusk-wallet/src/bin/command/history.rs

```diff
@@ -270,9 +283,31 @@ pub(crate) async fn moonlight_history(
                     amount += event_amount;
                 }
                 BlockData::PhoenixTransactionEvent(_) => {
+                    // This comes up in a phoenix to moonlight conversion.
+                    // In this conversion, a `PhoenixTransactionEvent`
+                    // and `ConvertEvent` are emitted.
```

Comment:
just like with `MoonlightTransactionEvent` the `PhoenixTransactionEvent` can also come up together with staking-related and/or other events.